### PR TITLE
LPD-31653 Update customExcludePaths test

### PR DESCRIPTION
**Pull request by fortunatomaldonado**
https://liferay.atlassian.net/browse/LPD-31653

I added a check for Save and Update button, and had the page wait till it fully loaded before page.evalute.

Let me know if there are any questions or comments about this.
Thank you!

### CODE REVIEW

File: modules/test/playwright/tests/frontend-js-spa-web/customExcludePaths.spec.ts [hunk 1]
```diff
@@ -63,6 +71,8 @@ test('@LPD-26354 Custom Exclude Path', async ({
 
 		await page.reload();
 
+		await page.locator('#content').waitFor({state: 'visible'});
```
[the remark this answers is not in the document]
Comment (fortunatomaldonado) [reply modules/test/playwright/tests/frontend-js-spa-web/customExcludePaths.spec.ts:74]: This is meant to check for a fully loaded page after a reload. I'm not sure if there is a better way to do this.

There is a utility to check after a save, just this is not happening in this situation.

[the remark this answers is not in the document]
Comment (dsanz) [reply modules/test/playwright/tests/frontend-js-spa-web/customExcludePaths.spec.ts:74]: ok if test works I don't see any reason not to move forward, so pls feel free to forward

**Comment by dsanz** [comment]
LGTM ;)

**Comment by dsanz** [comment]
ci:forward

**Pull request closed without merging**